src/app/containers/FrontPageMain/index.jsx

```diff
@@ -1,16 +1,24 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useContext } from 'react';
 import { shape } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { frontPageDataPropTypes } from '../../models/propTypes/frontPage';
 import { GhostWrapper, GridItemConstrainedLarge } from '../../lib/styledGrid';
+import { ServiceContext } from '../../contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import deepGet from '../../helpers/json/deepGet';
 
 const FrontPageMain = ({ frontPageData }) => {
+  const { brandName, translations } = useContext(ServiceContext);
+  const { home } = translations;
+
   const groups = deepGet(['content', 'groups'], frontPageData);
 
   return (
     <Fragment>
       <main role="main">
+        <VisuallyHiddenText as="h1">
+          <span lang="en">{brandName}</span> - {home}
```

Comment:
Is the lang attribute just around "BBC News"?

Reply:
Yes, it is.

Reply:
should we be preferring `en-GB`? articles seem to be using that in a similar context to this: #1861